includes/objectcache/MemcachedClient.php

```diff
@@ -874,10 +891,12 @@ protected function parseHost($host) {
 	 * @param $sock String: socket to mark as dead
 	 *
 	 * @access  private
+	 * @return false
```

Comment:
Can we have `void` here is the `false` here for purpose? (just for the sake of curiosity)